hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/fees/calculation/consensus/txns/UpdateMerkleTopicResourceUsageTest.java

```diff
@@ -96,66 +85,6 @@ void recognizesApplicableQuery() {
         assertFalse(subject.applicableTo(nonUpdateTopicTx));
     }
 
-    @Test
-    void getFeeThrowsExceptionForBadTxBody() {
-        final var mockTxnBody = mock(TransactionBody.class);
-        given(mockTxnBody.hasConsensusUpdateTopic()).willReturn(false);
-
-        Exception exception =
-                assertThrows(InvalidTxBodyException.class, () -> subject.usageGiven(null, sigValueObj, view));
-        assertEquals("consensusUpdateTopic field not available for Fee Calculation", exception.getMessage());
-
-        exception =
-                assertThrows(InvalidTxBodyException.class, () -> subject.usageGiven(mockTxnBody, sigValueObj, view));
-        assertEquals("consensusUpdateTopic field not available for Fee Calculation", exception.getMessage());
-
-        given(mockTxnBody.hasConsensusUpdateTopic()).willReturn(true);
-        exception = assertThrows(IllegalStateException.class, () -> subject.usageGiven(mockTxnBody, sigValueObj, null));
-        assertEquals("No StateView present !!", exception.getMessage());
-    }
-
-    @Test
-    void getFeeThrowsExceptionForBadKeys() throws InvalidKeyException, IllegalArgumentException {
-        final var txnBody = makeTransactionBody(
-                topicId,
-                defaultMemo,
-                JKey.mapJKey(adminKey),
-                JKey.mapJKey(submitKey),
-                IdUtils.asAccount("0.1.2"),
-                null,
-                null);
-        final var merkleTopic =
-                new MerkleTopic(defaultMemo, adminKey, submitKey, 0, new EntityId(0, 1, 2), new RichInstant(36_000, 0));
-        given(topics.get(EntityNum.fromTopicId(topicId))).willReturn(merkleTopic);
-        final var mockedJkey = mockStatic(JKey.class);
-        mockedJkey.when(() -> JKey.mapJKey(any())).thenThrow(new InvalidKeyException());
-
-        assertThrows(InvalidTxBodyException.class, () -> subject.usageGiven(txnBody, sigValueObj, view));
-        assertThat(
-                logCaptor.warnLogs(),
-                Matchers.contains(Matchers.startsWith("Usage estimation unexpectedly failed for")));
-        mockedJkey.close();
-    }
-
-    @Test
-    void updateToMissingTopic() throws InvalidKeyException, InvalidTxBodyException {
-        final var txBody = makeTransactionBody(
-                topicId,
-                defaultMemo,
-                JKey.mapJKey(adminKey),
-                JKey.mapJKey(submitKey),
-                IdUtils.asAccount("0.1.2"),
-                null,
-                null);
-        given(topics.get(EntityNum.fromTopicId(topicId))).willReturn(null);
-
-        final var feeData = subject.usageGiven(txBody, sigValueObj, view);
-
-        checkServicesFee(feeData, 0);
-        checkNetworkFee(feeData, 120, 0);
-        checkNodeFee(feeData, 120);
-    }
```

Comment:
Why are we getting rid of this test? Is it somehow related to the exceptions?

Reply:
Reverted. Thanks